OlinkAnalyze/R/Olink_heatmap.R

```diff
@@ -151,6 +153,26 @@ olink_heatmap_plot <- function(df,
                        values_from = NPX) %>%
     tibble::column_to_rownames('SampleID')
 
+  # Label columns according to the colnames argument
+  if(!colnames %in% c('Assay', 'oid', 'both')){
+    stop('colnames has to be \'Assay\', \'oid\', or \'both\'')
+  }
+  if(colnames == 'Assay'){
+    assays <- sub(pattern = '(.*)_.*', replacement = '\\1', x = colnames(npxWide)) %>%
+      as.data.frame() %>%
+      dplyr::rename('Assay' = 1) %>%
+      #If there are duplicate Assays (IL6 for instance), add a number to make the names unique
+      dplyr::group_by(Assay) %>%
+      dplyr::mutate(duplicateID = dplyr::row_number()) %>%
```

Comment:
I considered the same thing but got a little lazy :) One downside is also that it would require the panel column to be present in the data, or alternatively take this info from say D1141 (would introduce dependency on nexusr). Would it be worthwhile to add panel if the column exists, otherwise add a number?

Reply:
I thought it is safe to assume that the Panel column is present since we use it in some other functions and require it in the read_NPX for Explore data. Depending on internal documents or packages will not work... 
Might be not worth the effort to implement then.